compileFile.js

```diff
@@ -2,34 +2,40 @@ const { spawn } = require("child_process");
 const preferences = require("./preferencesHelper");
 const path = require("path");
 const locationHelper = require("./locationHelper");
+const config = require("./config")
+
+/**
+ * Get flags which needed for compile based on language
+ * @param {string} language programming language
+ * @param {string} filePath complete path to .c, .cpp or .py file
+ */
+function getFlags(language, filePath) {
+    const ext = config.extensions[language];
+    let flags = preferences().get("compilationFlags" + ext).split(" ");
+    if (flags[0] === "")
+        flags = [];
+
+    if (language === 'Python')
+        flags = ['-m', 'compileall', '-b', filePath, ...flags];
+    else {
+        // type of compiler ( g++ or gcc )
+        const outputLocation = locationHelper.getBinLocation(language, filePath);
```

Comment:
Let's make `g++` as `gpp` instead, here, and in other places. In the past I've used both, but using only one will help in maintaining consistency.

Reply:
I didn't understand what you mean. Do you want encrypt something?

Reply:
Where did you use `gpp` in the codebase?
I intended to preprocessing and compiling the source code first and create cached byte-code of it; then using the `pyc` to run test cases. AFAIK, `gpp` is just a preprocessor, not a compiler.

Reply:
You're right!